Cake.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+final class Cake
+{
+    public function __construct(private readonly int $candles)
+    {
+    }
+
+    public function serve(): void
+    {
+        if ($this->candles > 0) {
```

Comment:
Looks like the logic is inverted here. We should show an error when there are no candles, right?

Reply:
Done in 0ce64a4, thank you!